src/main/cpp/subsystems/HatchIntake.cpp

```diff
@@ -0,0 +1,45 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "subsystems/HatchIntake.h"
+static frc::DoubleSolenoid::Value kMotorEngage = frc::DoubleSolenoid::kForward;
+static frc::DoubleSolenoid::Value kMotorDisengage = frc::DoubleSolenoid::kReverse;
```

Comment:
Use an unnamed namespace instead of `static`. Also, use universal initialization syntax, and `constexpr`. This should be

```
namespace
{
  constexpr frc::DoubleSolenoid::Value kMotorEngage {frc::DoubleSolenoid::kForward};
  constexpr frc::DoubleSolenoid::Value kMotorDisengage {frc::DoubleSolenoid::kReverse};
}
```